Documentation/ApiOverview/Backend/BroadcastChannels.rst

```diff
@@ -75,29 +75,32 @@ Example code:
    });
 
 
-Hook into :php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['typo3/backend.php']['constructPostProcess']` to load a custom
-:php:`BackendController` hook that loads the event handler, e.g. via RequireJS.
+Hook into
+:php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['typo3/backend.php']['constructPostProcess']`
+to load a custom :php:`BackendController` hook that loads the event handlers
```

Comment:
```suggestion
to load a custom :php:`BackendController` hook that loads the event handler's
```

Above is a `require(['TYPO3/CMS/Backend/BroadcastService']` line. Just add a todo that this has to be changed the also (or do it now, if applicable) - or it is a completely separate task?